packages/venia-concept/src/components/ProductImageCarousel/__tests__/carousel.spec.js

```diff
@@ -41,7 +42,11 @@ const defaultProps = {
 };
 
 test('renders the Carousel component correctly w/ sorted images', () => {
-    const component = testRenderer.create(<Carousel {...defaultProps} />);
+    const component = testRenderer.create(
```

Comment:
In general, all new tests should use `createTestInstance()`, which wraps this method with `act()` [to ensure consistency](https://reactjs.org/docs/test-utils.html#act).